howtos/how-to-mark-a-conversation-as-answered/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you fill this out a bit more? Here's an example - I can help out if you want. https://github.com/talkjs/talkjs-examples/blob/master/react/basic-example/package.json

Specifically it'd be good to include a name and start script.

Reply:
Done